src/index.js

```diff
@@ -1,39 +1,38 @@
-import chroma from 'chroma-js';
-import def from '../default';
-import { Promise } from 'es6-promise';
-import es6denodeify from 'es6-denodeify';
-import extend  from 'extend';
-import fs from 'fs-extra';
-import { minify } from 'html-minifier';
-import path from 'path';
-import sassdocExtras from 'sassdoc-extras';
-import swig from './swig';
-
-const denodeify = es6denodeify(Promise);
-
-const copy = denodeify(fs.copy);
-const renderFile = denodeify(swig.renderFile);
-const writeFile = denodeify(fs.writeFile);
+import def from '../default'
+import { Promise } from 'es6-promise'
+import es6denodeify from 'es6-denodeify'
+import extend from 'extend'
+import fs from 'fs-extra'
+import { minify } from 'html-minifier'
+import path from 'path'
+import sassdocExtras from 'sassdoc-extras'
+import swig from './swig'
+
+const denodeify = es6denodeify(Promise)
+
+const copy = denodeify(fs.copy)
+const renderFile = denodeify(swig.renderFile)
+const writeFile = denodeify(fs.writeFile)
 
 const applyDefaults = ctx =>
   extend({}, def, ctx, {
     groups: extend(def.groups, ctx.groups),
-    display: extend(def.display, ctx.display),
```

Comment:
I’m not precious about it but if we want to keep it we need to rollback all the standard PRs.